### PR TITLE
Allow setting custom resync periods from helm

### PR DESCRIPTION
**Pull request by PsychoNineSix**
Following the implementation from https://github.com/SparebankenVest/azure-key-vault-to-kubernetes/issues/327

### CODE REVIEW

File: stable/akv2k8s/templates/controller-deployment.yaml [hunk 1]
```diff
@@ -84,6 +84,8 @@ spec:
           - "--v={{ template "controller.logLevel" . }}"
           - "--logging-format={{ template "controller.logFormat" . }}"
           - "--watch-all-namespaces={{ .Values.watchAllNamespaces }}"
+          - "--kubeResyncPeriod={{ .Values.kubeResyncPeriod }}"
+          - "--azureKeyVaultResyncPeriod={{ .Values.azureKeyVaultResyncPeriod }}"
         ports:
```
Comment (yyvess) [inline stable/akv2k8s/templates/controller-deployment.yaml:89]: Argument name was wrong must be =>

          - "--kube-resync-period={{ $.Values. kubeResyncPeriod }}"
          - "--azure-resync-period={{ $.Values.azureKeyVaultResyncPeriod }}"

**Review by yyvess — CHANGES_REQUESTED** [review]

File: stable/akv2k8s/templates/controller-deployment.yaml [hunk 2]
```diff
@@ -84,6 +84,8 @@ spec:
           - "--v={{ template "controller.logLevel" . }}"
           - "--logging-format={{ template "controller.logFormat" . }}"
           - "--watch-all-namespaces={{ .Values.watchAllNamespaces }}"
+          - "--kubeResyncPeriod={{ .Values.kubeResyncPeriod }}"
+          - "--azureKeyVaultResyncPeriod={{ .Values.azureKeyVaultResyncPeriod }}"
```
Comment (PsychoNineSix) [inline stable/akv2k8s/templates/controller-deployment.yaml:88]:
```suggestion
          - "--kube-resync-period={{ .Values.kubeResyncPeriod }}"
          - "--azure-resync-period={{ .Values.azureKeyVaultResyncPeriod }}"
```

File: stable/akv2k8s/templates/controller-deployment.yaml (hunk 1 above)
Comment (PsychoNineSix) [reply stable/akv2k8s/templates/controller-deployment.yaml:89]: Thank you, it's been changed!

**Comment by PsychoNineSix** [comment]
@181192 Without this it's currently not possible to use the new resync period, is it possible to get this PR at some point?

**Review by 181192 — APPROVED** [review]

**Comment by 181192** [comment]
@PsychoNineSix  released in chart version 2.2.2, thank you for the PR :blush:

**Comment by PsychoNineSix** [comment]
Thank you! :)

**Pull request merged**